web/src/i18n/locales/en/scheduleSummary.yaml

```diff
@@ -1,11 +1,12 @@
 title: Activity Schedule Summary
 helpText: >-
-  The Activity Schedule Summary shows all activities and milestones for the APD in a singular view.
+  See all activities and milestones for the APD together in one table.
```

Comment:
Example of 2nd vs 3rd person?

Reply:
This in particular is the kind of content I think is helpful to have in second person. The helptext/description at the top of each section is visible before the accordion is ever opened. The content can help someone understand what they would gain by opening the accordion(s) in that section. My own read is that direct address is more effective at that: it pushes more toward using active voice, it gives people an instruction to follow, etc. So I tried to work these descriptions into that form.

(That paragraph was a little convoluted to write because `helpText` is the label we use for a couple different kinds of content, depending on whether it's inside or outside the accordion. We're fixing it for within the accordion, but the shift we'll be making does raise an issue for what to call this text outside of it. This specific example is outside the accordion. No action for right now but cc @hbillings for visibility; I think this issue might be up her alley.)

That said, if there's something incongruous about this choice with the way state staff are used to reading CMS content, it's not set in stone. I think it's important to keep these elements consistent _with each other_ to give people recognizable patterns and make it easy to read the same kind of content the same way in different places.

Reply:
Per your comment at the end of the PR, I'm going to backlog this question and not treat it as a show-stopper for now.

Reply:
Great to understand context/use. Thank you for the detailed response!